ALVRClient.xcodeproj/project.pbxproj

```diff
@@ -367,6 +397,7 @@
 				DEVELOPMENT_ASSET_PATHS = "\"ALVRClient/Preview Content\"";
 				ENABLE_PREVIEWS = YES;
 				EXCLUDED_ARCHS = x86_64;
+				GCC_OPTIMIZATION_LEVEL = 0;
```

Comment:
I don't think it realistically makes a difference, since we have no C/C++/ObjC code, but this build setting should be removed and left as the default.

Reply:
updated